src/mapml-viewer.js

```diff
@@ -986,7 +989,256 @@ export class HTMLMapmlViewerElement extends HTMLElement {
       }
     });
   }
+  matchMedia(query) {
+    // useful features for maps: prefers-color-scheme, prefers-lang, projection, zoom, extent
+    const parsedQuery = parseMediaQueryList(query);
+
+    // less obviously useful: aspect-ratio, orientation, (device) resolution, overflow-block, overflow-inline
+
+    const map = this;
+    const features = {
+      'prefers-lang': {
+        type: 'discrete',
+        get values() {
+          return [navigator.language.substring(0, 2)];
+        }
+      },
+      'map-projection': {
+        type: 'discrete',
+        get values() {
+          return [map.projection.toLowerCase()];
+        }
+      },
+      'map-zoom': {
+        type: 'range',
+        valueType: 'integer',
+        canBeNegative: false,
+        canBeZero: true,
+        get extraValues() {
+          return {
+            min: 0,
+            max: map.zoom
+          };
+        }
+      },
+      'map-top-left-easting': {
+        type: 'range',
+        valueType: 'integer',
+        canBeNegative: true,
+        canBeZero: true,
+        get values() {
+          return [Math.trunc(map.extent.topLeft.pcrs.horizontal)];
+        }
+      },
+      'map-top-left-northing': {
+        type: 'range',
+        valueType: 'integer',
+        canBeNegative: true,
+        canBeZero: true,
+        get values() {
+          return [Math.trunc(map.extent.topLeft.pcrs.vertical)];
+        }
+      },
+      'map-bottom-right-easting': {
+        type: 'range',
+        valueType: 'integer',
+        canBeNegative: true,
+        canBeZero: true,
+        get values() {
+          return [Math.trunc(map.extent.bottomRight.pcrs.horizontal)];
+        }
+      },
+      'map-bottom-right-northing': {
+        type: 'range',
+        valueType: 'integer',
+        canBeNegative: true,
+        canBeZero: true,
+        get values() {
+          return [Math.trunc(map.extent.bottomRight.pcrs.vertical)];
+        }
+      },
+      'prefers-color-scheme': {
+        type: 'discrete',
+        get values() {
+          return [
+            window.matchMedia('(prefers-color-scheme: dark)').matches
```

Comment:
This unfortunately does not seem to work for me on chrome, when changing the mode
![image](https://github.com/user-attachments/assets/887d8299-8fd3-4b7a-8504-cd7827d2710b). `window.matchMedia('(prefers-color-scheme: dark)').matches` also does not work

Reply:
Aha we went through that pain too!  You have to go into devtools and click on the rendering panel:

![image](https://github.com/user-attachments/assets/57b4b5e0-675c-4eae-bb88-577982138ea0)
and then select 'Emulate CSS media feature prefers-color-scheme' (select dark):
![image](https://github.com/user-attachments/assets/86d3e0a0-d9f1-4fb7-8f6f-c2835e05b3ce)